src/main/kotlin/no/nav/syfo/dialogmote/DialogmoteService.kt

```diff
@@ -698,7 +701,7 @@ class DialogmoteService(
             referatUUID = brevUuid
         )
 
-        if (motedeltakerArbeidstakerVarsel == null && referat == null) {
+        if ((motedeltakerArbeidstakerVarsel == null && referat == null) || (referat != null && !referat.ferdigstilt)) {
```

Comment:
Dette boolske utrykket er veldig langt nå og da bir det vanskelig å lese. Kanskje legge på en navngitt variabel eller to her?
Samme for linje 716.

Reply:
Fikset